lib/facility_factory.rb

```diff
@@ -0,0 +1,39 @@
+class FacilityFactory
+
+  def create_or_facility(facility)
+    facility = facility.slice(:title,:location_1, :phone_number)
+    facility[:name] = facility.delete(:title)
```

Comment:
This is really smart! I like that you're adding in the key you need. You could also just do `facility[:name] = facility[:title]`, no need to actually delete the key, we just need the value.